malasakit-django/pcari/templates/quantitative-questions.html

```diff
@@ -47,16 +47,23 @@
 
 {% block content %}
   <ol>
-    {% for qid, prompt, left_text, right_text in questions %}
+    {% for qid, prompt, type, minval, maxval, left_text, right_text in questions %}
       <li>
         <p class="prompt">{% trans prompt %}</p>
         <span class="leftlabel">{% trans left_text %}</span>
-        <input id="rating-{{ qid }}" type="range" value="5" min="1" max="10">
+        {% ifequal type "range" %}
+          <input name="{{ qid }}" type="range" value="{{ minval }}"
+                 min="{{ minval }}" max="{{ maxval }}">
+        {% endifequal %}
+        {% ifequal type "number" %}
+          <input name="{{ qid }}" type="number" style="max-width:200px;"
```

Comment:
This needs the `id` attribute as well (`rating-<qid>`) so that the JavaScript works.